src/ECPublicKeyTrait.php

```diff
@@ -10,12 +10,23 @@
     // Stored base64-encoded
     private $pubKey = '';
 
-    // Binary string of public key
+    /**
+     * @deprecated Methods that return binary values are suffixed with "binary".
+     * @return string Binary string of public key
+     */
     public function getPublicKey(): string
     {
         return base64_decode($this->pubKey);
```

Comment:
Since this method is being deprecated, I think it's preferable to `return $this->getPublicKeyBinary();` instead of using the same implementation - it makes it clearer that the method is being renamed.

It also may be worth firing a `trigger_error(E_USER_DEPRECATED, ...);` as well, to be proactive. Thoughts?

Reply:
Not sure about `trigger_error(E_USER_DEPRECATED, ...)`. Doesn't sound too different from removing the method altogether :p Certain static analysis tools such as scrutinizer warn of the use of deprecated functions, I wonder if PHP Storm does the same.

Although the advantage of `trigger_error(E_USER_DEPRECATED, ...)` is that the error and the reasons / fix is made explicit to the developer but you still give him the possibility to ignore it. I suppose I should do it then, you confirm?